fixtures/stairville/mini-stage-par-rgb-ww.json

```diff
@@ -0,0 +1,254 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Stage Par RGB WW",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Luke Nelson"],
+    "createDate": "2022-12-28",
+    "lastModifyDate": "2022-12-28",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2022-12-28",
+      "comment": "created by Q Light Controller Plus (version 4.12.4a)"
+    }
+  },
+  "links": {
+    "productPage": [
+      "https://www.thomann.de/gb/stairville_mini_stage_par_7x4w_rgb_ww.htm"
+    ],
+    "video": [
+      "https://video1.thomann.de/vidiot/02591c1c/video_i6194p10_yd59vqpa.mp4",
+      "https://www.youtube.com/watch?v=wm-QSmre8Iw"
+    ],
+    "manual": [
+      "https://images.static-thomann.de/pics/atg/atgdata/document/manual/406532_c_406531_406532_r4_en_online.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [195, 140, 230],
+    "weight": 1.9,
+    "power": 30,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [30, 30]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Fixed colour pattern": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "Effect",
+          "effectName": "No fixed colour and movement pattern"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "Effect",
+          "effectName": "One of 41 preprogrammed automatic shows (macros), channels 6 and 7 without function"
```

Comment:
Which show at which DMX values?

Reply:
Unfortunately, neither the values nor the shows are documented.